packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -1170,6 +1201,24 @@ void main() {
     ProcessManager: () => FakeProcessManager.any(),
   });
 
+  testUsingContext('wasm mode selects skwasm renderer by default', () async {
+    final RunCommand command = RunCommand();
+    await expectLater(() => createTestCommandRunner(command).run(<String>[
+      'run',
+      '-d chrome',
+      '--wasm',
+    ]), throwsToolExit());
+
+    final DebuggingOptions options = await command.createDebuggingOptions(false);
```

Comment:
This test is a little bit wacky, we wait for run to fail with `command`, but then we can reliably read the debugging options off of it? :S